x-pack/test/functional/services/machine_learning/api.ts

```diff
@@ -8,10 +8,24 @@ import expect from '@kbn/expect';
 import { isEmpty } from 'lodash';
 import { FtrProviderContext } from '../../ftr_provider_context';
 
+export enum JobState {
```

Comment:
There are const enums in ML which could be used instead of defining your own.
https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/ml/common/constants/states.ts

If it's not possible to import from ML like that, i'd suggest you copy the style and make these constants upper case.

Reply:
Great hint, thanks! Changed it in f32ca31